charters/DataProcessingService/charter.md

```diff
@@ -0,0 +1,65 @@
+# [Data Processing Service](mailto:pipelines-team@data.humancellatlas.org)
+
+## Description
+
+The Data Processing Service is a service that executes data processing pipelines on DCP data in the Data Storage Service. It can be used in any instance of the Data Coordination Platform (DCP). 
+
+## Objectives
+
+The Data Processing Service aims to develop the infrastructure to run pipelines reliably and at-scale for the DCP. The team working on the service will also operate the infrastructure for the Human Cell Atlas. 
+
+## In-scope
+
+* Creating and maintaining the Data Processing Service  
+
+### Operations
+
+* Operating the instance of the HCA DCP Data Processing Service
+* Initiation and management of the execution of all data processing pipelines that run in the HCA DCP  
+* Establishing and operating automated practices around engineering testing of pipelines in the HCA DCP  
+
+### Documentation
+
+* Creating and outputting metadata on the transformations done to the data in accordance with the HCA DCP Metadata Schema  
+* Documentation on the Data Processing Service and guides on using the service  
+* Creating documentation and scripting around the deployment of the Data Processing Service
+
+## Out-of-scope
+
+* Standing up or operating other instances of the Data Processing Service for other projects  
+
+## Milestones and Deliverables
+
+__2018:__ Create operational and secure infrastructure, add ability to throttle and queue  
+__2019:__ Add ability to reanalyze data, notification filtering to save costs, improve testing, logging, and deployment  
+__2020:__ Generalized and reusable service infrastructure  
+
+## Roles
+
+### Project Lead
+
+[Timothy Tickle](mailto:ttickle@broadinstitute.org)  
+
+### Product Owner
+
+[Kylee Degatano](mailto:kdegatano@broadinstitute.org)  
+
+### Technical Lead
+
+[Saman Ehsan](mailto:sehsan@broadinstitute.org)  
+[Rex Wang](mailto:chengche@broadinstitute.org)  
+
+## Communication
+
```

Comment:
Mint team isn't named mint team because of colored boxes, it's just our team name at the Broad. We could make separate channels for user facing purposes i.e. #dataprocessingservice , but the #mintteam channel is meant for our inter-team discussion.

Reply:
This is interesting because I think a lot of the teams named their channels for their product and but we did using our internal team name (the color naming is a false positive as Kylee mentioned; we actually have completely unrelated internal green and red teams as well). I can definitely see it would be difficult to know which channel to go to in the slack (this charter of course does help and guide). As Kylee mentioned, happy to make a #dataprocessingservice channel and list it here as an outward facing channel.

Reply:
Yes - I'd recommend renaming the secondary analysis one though, not creating a new one.

Reply:
Adding updated names in the charter, will update actual slack around Friday.